package.json

```diff
@@ -22,7 +22,8 @@
     "build": "rm -rf build && babel src -d build",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "db:migrate": "sequelize db:migrate:undo:all && sequelize db:migrate",
-    "db:migrate:testdb": "NODE_ENV=test npm run db:migrate"
+    "db:migrate:testdb": "NODE_ENV=test npm run db:migrate",
+    "lint": "node_modules/eslint/bin/eslint.js src --fix"
```

Comment:
1. Kindly include this after the nyc section of your package.json file
"husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  },
  "lint-staged": {
    "*.{js,json,css,md}": [
      "prettier --write",
      "git add"
    ]
  },

2. Also, please add these to your npm scripts:
"check-staged": "lint-staged",
"prettier": "prettier --write '**/*.{js,jsx}'",

They are necessary measure to prevent you from committing if any eslint errors are present in your code

Reply:
NIce!!!

Reply:
this has been resolved